### PR TITLE
blog: devxconf.org wrap

### PR DESCRIPTION
**Pull request by ghuntley**
<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/548"></a>

### CODE REVIEW

File: src/contents/blog.ts [hunk 1]
```diff
@@ -156,4 +156,14 @@ export const authors: { [idx: string]: Author } = {
...
   },
+  christinfrohne: {
+    name: "Christin Frohne",
+    socialProfiles: {
+      github: "ChristinFrohne",
+      twitter: "christinfrohne",
+      linkedin: "christin-frohne-91a4b0173",
+    },
+    description:
+      "Christin is our Marketing Manager at Gitpod. She enjoys yoga, climbing, being in the nature, building furniture out of recycled materials",
```
Comment (ghuntley) [inline src/contents/blog.ts:167]: @ChristinFrohne I took some liberties here and crafted verbiage based upon Notion > Gitpodders. Let me know if you want this changed/updated.

File: src/routes/blog/devxconf-wrap.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+---
+author: christinfrohne, ghuntley
+date: Tuesday, 1 Jun 2021 11:00:00 UTC
+excerpt: TBA
```
Comment (ghuntley) [inline src/routes/blog/devxconf-wrap.md:4]: 🎗 resolve before publishing

File: src/routes/blog/devxconf-wrap.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+image: teaser.jpg
+slug: devxconf-wrap
+subtitle:
+teaserImage: teaser.jpg
+title: DevX Conf Wrap
```
Comment (ghuntley) [inline src/routes/blog/devxconf-wrap.md:9]: 💭 improve title to include mention of how funds are distributed

File: src/routes/blog/devxconf-wrap.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+Only a fraction of funds have been distributed at this stage as a industry we haven't documented and established clear ways where companies are able give open-source projects money.
+
+Sending hand crafted emails (assuming an email address is publically available 😢) asking projects these questions isn't scalable and most companies would not do this manual grunt work. 😉
+
+If you are the stweard of one of the above open-source projects, then there's one simple, easy step you can do to enable Gitpod and other companies to provide financial support...
+
+- 1. Go sign up to [GitHub Sponsors](https://github.com/sponsors) or [Open Collective](https://www.opencollective.com)
+- 2. Send an email to <a href="mailto:contact@gitpod.io?subject=Distribution of DevX Open-source Funding">contact@gitpod.io</a>.
+- 3. Nominate where funding should be directed (if you wish to divert the funds to a cause that helps improve diviserty such as [Outreachy](https://www.outreachy.org/) thats possible!)
```
Comment (ghuntley) [inline src/routes/blog/devxconf-wrap.md:110]: 💭 add 4. document this information.

File: src/routes/blog/devxconf-wrap.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
+---
+author: christinfrohne, ghuntley
+date: Tuesday, 1 Jun 2021 11:00:00 UTC
+excerpt: TBA
+image: teaser.jpg
+slug: devxconf-wrap
+subtitle:
+teaserImage: teaser.jpg
```
Comment (ghuntley) [inline src/routes/blog/devxconf-wrap.md:8]: 🎗 needs header image

File: src/routes/blog/devxconf-wrap.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+If you are the stweard of one of the above open-source projects, then there's one simple, easy step you can do to enable Gitpod and other companies to provide financial support...
+
+- 1. Go sign up to [GitHub Sponsors](https://github.com/sponsors) or [Open Collective](https://www.opencollective.com)
+- 2. Publically document the process of how companies can support your project.
+- 3. Send an email to <a href="mailto:contact@gitpod.io?subject=Distribution of DevX Open-source Funding">contact@gitpod.io</a>.
+- 4. Nominate where funding should be directed (if you wish to divert the funds to a cause that helps improve diviserty in open-source such as [Outreachy](https://www.outreachy.org/) thats possible!)
+
+# Thanks
+
+Big shout out to Vercel for creating the [Event Starter Kit](https://vercel.com/virtual-event-starter-kit) and making it open-source. It facilitated our lives a lot and basically gave us the confidence to organize the conference in the first place. Thanks a lot! 🙇‍♀️ 🙇‍♂️
```
Comment (ChristinFrohne) [inline src/routes/blog/devxconf-wrap.md:115]: This feels a bit out of context now. Suggestion:

"Thanks to everyone who participated in DevX Conf and who took a vote to support these great open-source projects.  
A great shout-out also goes to Vercel..."

File: src/routes/blog/devxconf-wrap.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+A big thanks also goes to the whole team organizing the event including Johannes Landgraf, Nisar Hassan Naqvi, Geoffrey Huntley, and Mike Nikles 🙌
```
Comment (ChristinFrohne) [inline src/routes/blog/devxconf-wrap.md:117]: Should we leave you out here, Geoff? As you are co-authoring this blog post and it's written in both our names? :) 

"A big thanks also goes to the [rest of the] team... "

**Review by ChristinFrohne — COMMENTED** [review]
Great blog post @ghuntley 👏 
Thank you for opening the PR

I added some comments mostly regarding the layout inline :)

File: src/routes/blog/devxconf-wrap.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
+---
+author: christinfrohne, ghuntley
+date: Thursday, 3 Jun 2021 11:00:00 UTC
+excerpt: As part of Gitpod’s Open-Source Sustainability Fund initiative attendees of DevX Conf were able to decide where $10,000 USD of funds (the profits from DevX Conf) were to be distributed. Here's the breakdown how the fund was split and a retrospective of Gitpod running our first ever conference.
+image: teaser.jpg
+slug: devxconf-wrap
+subtitle:
+teaserImage: teaser.jpg
+title: DevX Conf wrap and the distribution of $10,000USD in Open Source funding
```
[the remark this answers is not in the document]
Comment (ghuntley) [reply src/routes/blog/devxconf-wrap.md:9]: 👍 DevX Conf wrap & distributing $10k of open-source funding

File: src/routes/blog/devxconf-wrap.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+[![I'm very glad developer experience is picking up so much steam that there's now a dedicated conference for it: http://devxconf.org. Not only that, Gitpod is donating all the commercial proceeds from sponsors to open source projects they rely on!! Exploding 🤯👏](../../../static/images/blog/devxconf-wrap/tweet-mxstbr.png)](https://twitter.com/mxstbr/status/1385511937845387269)
```
[the remark this answers is not in the document]
Comment (ghuntley) [reply src/routes/blog/devxconf-wrap.md:20]: Documented in Notion > DCS > Rounded Corners and https://www.figma.com/file/8qUX9HmSTFxdkmtlr16XKP/Untitled?node-id=0%3A1 is upstream file.

File: src/routes/blog/devxconf-wrap.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+- **1. what exactly is digital infrastructure?**
+- **2. what tools are relied upon by developers everyday that are built and maintained almost exclusively by unpaid volunteers?**.
+
+After some dialectical thinking the answer became clear: projects built by unpaid voltuneers in the category of [linters](<https://en.wikipedia.org/wiki/Lint_(software)>) and [language server protocol](https://langserver.org/) implementations.
+
+Over the last month Gitpod has been emailing maintainers and distributing the funds.
+
+[![🤯👏](../../../static/images/blog/devxconf-wrap/shellcheck-email.png)](https://github.com/koalaman/shellcheck)
+
+[![🤯👏](../../../static/images/blog/devxconf-wrap/shellcheck-sponsorship.png)](https://github.com/koalaman/shellcheck)
```
[the remark this answers is not in the document]
Comment (ghuntley) [reply src/routes/blog/devxconf-wrap.md:68]: Played with the image placement and removed one embed.

File: src/routes/blog/devxconf-wrap.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+<br/>
+
+Only a fraction of funds have been distributed at this stage because as a industry we haven't documented and established clear ways where companies are able give open-source projects money.
+
+Sending hand crafted emails (assuming an email address is publically available 😢) asking projects these questions isn't scalable and most companies would not do this manual grunt work. 😉
+
+If you are the stweard of one of the above open-source projects, then there's one simple, easy step you can do to enable Gitpod and other companies to provide financial support...
+
+- 1. Go sign up to [GitHub Sponsors](https://github.com/sponsors) or [Open Collective](https://www.opencollective.com)
```
[the remark this answers is not in the document]
Comment (ghuntley) [reply src/routes/blog/devxconf-wrap.md:108]: Raised at #561

**Pull request merged**